### ISSUE TITLE
Undefined offset 1 when editing Company

### ISSUE THREAD

**Issue by tbeachy** [iomad/iomad#1457]
IOMAD_38_STABLE

When editing a company and debugging turned on, displays this error:
( ! ) Notice: Undefined offset: 1 in /home/oe9us12ub0cm/public_html/blocks/iomad_company_admin/classes/forms/company_edit_form.php on line 157
Call Stack
#	Time	Memory	Function	Location
1	0.0127	414008	{main}( )	.../company_edit_form.php:0
2	0.1142	5508776	block_iomad_company_admin\forms\company_edit_form->__construct( )	.../company_edit_form.php:263
3	0.1142	5510112	block_iomad_company_admin\forms\company_edit_form->__construct( )	.../company_edit_form.php:53
4	0.1265	6303560	block_iomad_company_admin\forms\company_edit_form->definition( )	.../formslib.php:214

**Comment by turf212** [comment]
That line relates to email templates which may or may not have been saved on the system:

            `$mform->addElement('select', 'emailtemplate', get_string('applyemailtemplate', 'block_iomad_company_admin', $emailtemplates[$this->previousemailtemplateid]), $emailtemplates);`

I've tested this on a clean installation.  Editing the company and creating a new company doesn't throw an errors on the screen (I have full developer debug on).  When does this happen?  When you are creating a new company?  A child company?  Editing an existing company?  An existing child company?

**Comment by tbeachy** [comment]
This happens when editing an existing company.  I have no child companies at this point and only two top-level companies.  No licensing, no saved email templates.  I did have a saved template at one point when testing that I have since deleted.  I just tried it again, and can still see it in full dev debug. 
I don't know if it's causing any problems, but noticed it when I was working on the other email issue I had that you resolved.
I don't have any ideas for other tests to help pinpoint the problem at this point, unless you do?
Thanks.
